CallAutomation_OutboundCalling/README.md

```diff
@@ -0,0 +1,53 @@
+|page_type|languages|products
+|---|---|---|
+|sample|<table><tr><td>Typescript</tr></td></table>|<table><tr><td>azure</td><td>azure-communication-services</td></tr></table>|
+
+# Call Automation - Quick Start Sample
+
+This sample application shows how the Azure Communication Services  - Call Automation SDK can be used to build IVR related solutions. 
+It makes an outbound call to a phone number, performs DTMF recognition, plays a different audio message based on the key pressed by the callee and hangs-up the call. 
+This sample application configured for accepting tone 1 (tone1), 2 (tone2) , If the callee pressed any other key than expected, the call will be disconnected.
+This sample application is also capable of making multiple concurrent outbound calls. The application is a web-based application built on Express.js framework.
+
+# Design
+
+![design](./data/OutboundCallDesign.png)
+
+## Prerequisites
+
+- Create an Azure account with an active subscription. For details, see [Create an account for free](https://azure.microsoft.com/free/)
+- [Visual Studio (2022 v17.4.0 and above)](https://visualstudio.microsoft.com/vs/)
+- [Node.js](https://nodejs.org/en/download) installed
+- Create an Azure Communication Services resource. For details, see [Create an Azure Communication Resource](https://docs.microsoft.com/azure/communication-services/quickstarts/create-communication-resource). You will need to record your resource **connection string** for this sample.
+- Get a phone number for your new Azure Communication Services resource. For details, see [Get a phone number](https://learn.microsoft.com/en-us/azure/communication-services/quickstarts/telephony/get-phone-number?tabs=windows&pivots=programming-language-csharp)
+- Enable Visual studio dev tunneling for local development. For details, see [Enable dev tunnel] (https://learn.microsoft.com/en-us/connectors/custom-connectors/port-tunneling)
+	- To enable dev tunneling, Click `Tools` -> `Options` in Visual Studio 2022
+	- In the search bar type tunnel, Click the checkbox under `Environment` -> `Preview Features` called `Enable dev tunnels for Web Application`
+	- ![EnableDevTunnel](./data/EnableDevTunnel.png) 
+	- Login into your account under `Dev Tunnels` -> `General`
+	- ![LogInDevTunnel](./data/AddAccountForTunnel.png) 
```

Comment:
Remove above.
Optionally add (or leave it out, it will be mentioned in the quickstart documentation):
### Setup and host your Azure DevTunnel

[Azure DevTunnels](https://learn.microsoft.com/en-us/azure/developer/dev-tunnels/overview) is an Azure service that enables you to share local web services hosted on the internet. Use the commands below to connect your local development environment to the public internet. This creates a tunnel with a persistent endpoint URL and which allows anonymous access. We will then use this endpoint to notify your application of calling events from the ACS Call Automation service.

```bash
devtunnel create --allow-anonymous
devtunnel port create -p 8080
devtunnel host
```